### ISSUE TITLE
Evaluate terminal find screen reader support

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#634]
It's been suggested in https://github-com/microsoft/vscode/-/149882-issuecomment-1134946927 that find doesn't work well when using a screen reader.

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.8. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.8. does meet threshold
